WordPress/Classes/Networking/ReaderPostServiceRemote.m

```diff
@@ -979,6 +983,52 @@ - (NSString *)formatVideoPress:(NSString *)string
     return mstr;
 }
 
+- (NSString *)resizeGalleryImageURLsForContent:(NSString *)content isPrivateSite:(BOOL)isPrivateSite
+{
+    if (!content) {
+        return @"";
+    }
+
+    NSMutableString *mcontent = [content mutableCopy];
+
+    static NSRegularExpression *regexGalleryImages;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        regexGalleryImages = [NSRegularExpression regularExpressionWithPattern:@"<img[^>]*data-orig-file[^>]*\\/>" options:NSRegularExpressionCaseInsensitive error:nil];
```

Comment:
Looking at the pattern `<img[^>]*data-orig-file[^>]*\\/>` a bit.  The `\\` near the end is superfluous. Its not affecting the matching as far as I can tell, but maybe we could remove it for a slightly optimized pattern.